src/protocol/thread/value.ts

```diff
@@ -152,6 +152,11 @@ export class ValueFront {
     return (this.hasPreview() && this._object!.preview!.promiseState) || null;
   }
 
+  previewProxyState() {
+    // Older recordings did not set proxyState so this could be null.
+    return (this.hasPreview() && this._object!.preview!.proxyState) || null;
```

Comment:
Small nit, but what do you think about doing: `return this._object?.preview?.proxyState || null`.

Reply:
Meh could go either way, tbh this is more readable!

Reply:
This was essentially a copy-paste from the function above this, so probably in favor of leaving it.